js/pivx_shielding.js

```diff
@@ -353,3 +399,28 @@ export class UTXO {
     this.script = script;
   }
 }
+
+export class ShieldDB {
```

Comment:
I don't really like this name, this is not a database, it shouldn't be called DB